_includes/footer.html

```diff
@@ -45,3 +45,14 @@ <h5>Our Social Ocean</h5>
     </div>
   </div>
 </footer>
+
+<script src="https://unpkg.com/mermaid@8.9.3/dist/mermaid.min.js"></script>
+<script>
+  $(document).ready(function () {
+    mermaid.initialize({
+      startOnLoad:true,
+      theme: "default",
+    });
+    window.mermaid.init(undefined, document.querySelectorAll('.language-mermaid'));
+  });
+</script>
```

Comment:
This means that the JS in `mermaid.html` will be added in the HTML body only when we have the front matter `mermaid: true` variable. This will make the pages without the chart load faster.

```suggestion
{% if page.mermaid %}
  {% include mermaid.html %}
{% endif %}
```